modules/utxo-lib/test/bitgo/wallet/util.ts

```diff
@@ -118,7 +118,10 @@ export function mockWalletUnspent<TNumber extends number | bigint>(
       BigInt(value),
       network
     );
-    const unspent = isSegwit(chain) ? fromOutput(prevTransaction, vout) : fromOutputWithPrevTx(prevTransaction, vout);
+    const unspent =
```

Comment:
Is there also some test code (or fixtures) that are creating prevtxs for zcash unnecessarily?

Reply:
AFAIK no